src/import.ts

```diff
@@ -0,0 +1,203 @@
+      .then((resp: GetAppCustomizeResp) => {
+        console.log(m("M_DownloadUploadedFile"));
+        return downloadCustomizeFiles(
+          kintoneApiClient,
+          appId,
+          options.destDir,
+          resp
+        );
+      });
+  } catch (e) {
+    const isAuthenticationError = e instanceof AuthenticationError;
+    retryCount++;
+    if (isAuthenticationError) {
+      throw new Error(m("E_Authentication"));
+    } else if (retryCount < Constans.MAX_RETRY_COUNT) {
+      await wait(1000);
+      console.log(m("E_Retry"));
+      await importCustomizeSetting(
+        kintoneApiClient,
+        manifest,
+        { retryCount },
+        options
+      );
+    } else {
+      throw e;
+    }
+  }
+}
+
+function exportAsManifestFile(
+  appId: string,
+  destRootDir: string,
+  resp: GetAppCustomizeResp
+): GetAppCustomizeResp {
+  const toNameOrUrl = (destDir: string) => (f: CustomizeFile) => {
+    if (f.type === "FILE") {
+      return `${destDir}/${f.file.name}`;
+    } else {
+      return f.url;
+    }
+  };
+
+  const desktopJs: CustomizeFile[] = resp.desktop.js;
+  const desktopCss: CustomizeFile[] = resp.desktop.css;
+  const mobileJs: CustomizeFile[] = resp.mobile.js;
+
+  const customizeJson: CustomizeManifest = {
+    app: appId,
+    scope: resp.scope,
+    desktop: {
+      js: desktopJs.map(toNameOrUrl(`${destRootDir}/desktop/js`)),
+      css: desktopCss.map(toNameOrUrl(`${destRootDir}/desktop/css`))
+    },
+    mobile: {
+      js: mobileJs.map(toNameOrUrl(`${destRootDir}/mobile/js`))
+    }
+  };
+
+  if (!fs.existsSync(`${destRootDir}`)) {
+    mkdirp.sync(`${destRootDir}`);
+  }
+  fs.writeFileSync(
+    `${destRootDir}/customize-manifest.json`,
+    JSON.stringify(customizeJson, null, 4)
+  );
+  return resp;
+}
+
+async function downloadCustomizeFiles(
+  kintoneApiClient: KintoneApiClient,
+  appId: string,
+  destDir: string,
+  { desktop, mobile }: GetAppCustomizeResp
+): Promise<any> {
+  const desktopJs: CustomizeFile[] = desktop.js;
+  const desktopCss: CustomizeFile[] = desktop.css;
+  const mobileJs: CustomizeFile[] = mobile.js;
+
+  [
+    `${destDir}${sep}desktop${sep}js${sep}`,
+    `${destDir}${sep}desktop${sep}css${sep}`,
+    `${destDir}${sep}mobile${sep}js${sep}`
+  ].forEach(path => mkdirp.sync(path));
+
+  const desktopJsPromise = desktopJs.map(
+    downloadAndWriteFile(kintoneApiClient, `${destDir}${sep}desktop${sep}js`)
+  );
+  const desktopCssPromise = desktopCss.map(
+    downloadAndWriteFile(kintoneApiClient, `${destDir}${sep}desktop${sep}css`)
+  );
+  const mobileJsPromise = mobileJs.map(
+    downloadAndWriteFile(kintoneApiClient, `${destDir}${sep}mobile${sep}js`)
+  );
+  return desktopJsPromise.concat(desktopCssPromise).concat(mobileJsPromise);
+}
+
+function downloadAndWriteFile(
+  kintoneApiClient: KintoneApiClient,
+  destDir: string
+): (f: CustomizeFile) => void {
+  return async f => {
+    if (f.type === "URL") {
+      return;
+    }
+    kintoneApiClient
```

Comment:
You have to use `await` or return the value of `kintoneApiClient.downloadFile().then()`